app/views/employee/index.html.erb

```diff
@@ -0,0 +1,64 @@
+<div class="main-container">
+  <div class="add-container">
+    <div class="add-button">
+      <span> + </span>
+    </div>
+    <div class="add-text-container">
+      <h1>Add new employee</h1>
+    </div>
+  </div>
+  <div class="table-container">
+    <section class="header-table">
+      <div>
+        <ul>
+          <li class="header-table-text">
+            <p>Email</p>
+          </li>
+          <li class="header-table-text">
+            <p>Name</p>
+          </li>
+          <li class="header-table-text">
+            <p>Position</p>
+          </li>
+          <li class="header-table-text">
+            <p>Number</p>
+          </li>
+          <li class="header-table-text">
+            <p>Private Number</p>
+          </li>
+          <li class="header-table-text">
+            <p>Actions</p>
+          </li>
+        </ul>
+      </div>
+    </section>
```

Comment:
If you have a `section` label, what is the reason for containing the 'ul' in a 'div'?